README.md

```diff
@@ -1,4 +1,30 @@
+
 # Tacoma Web Design and SEO
+
 ## Web Design Starter Template
 
-This template is used for the starting of all web design projects for Tacoma Web Design and SEO. Further documentation can be found internally and through this README https://github.com/CodeStitchOfficial/Intermediate-Website-Kit-SASS.
\ No newline at end of file
+  
+
+This template is used for the starting of all web design projects for Tacoma Web Design and SEO. Further documentation can be found internally and through this README https://github.com/CodeStitchOfficial/Intermediate-Website-Kit-SASS.
+
+## Optimization Changes
+
+
+- [Auto Image Optimization](#auto-image-optimization)
```

Comment:
can you add more to this ReadMe. For each of these the following two pieces of information would be helpful
1. why this is important
2. how to do it

Reply:
Hello Collin @gamrothc  Thanks for reviewing this. I've added and updated the readme , also updated other optimization why they are added and how it works. Some doesn't have "how to do" it as it doesn't require any additional configuration for the dev. Thanks